doc/cascadia/profiles.schema.json

```diff
@@ -54,6 +54,12 @@
         }
       ]
     },
+    "BellSound": {
+      "default": "",
+      "description": "Sets the file location of the sound played when the application emits a BEL character. If the path is invalid no sound will be played. This property also accepts an array of sounds and the terminal will pick one at random.",
+      "pattern": "",
```

Comment:
@carlos-zamora would we want regex for a local file or an empty string is alright?

Reply:
eh, empty string is fine. I think that's what we did with `backgroundImage` which is a similar idea.

Reply:
In fact, I suggest outright removing the pattern property here. I don't think it does anything (or at least anything valuable here?)

Reply:
Removed the pattern variable